ui/app/components/send/send-v2-container.js

```diff
@@ -36,23 +40,24 @@ function mapStateToProps (state) {
 
     primaryCurrency = selectedToken.symbol
 
-    tokenToUSDRate = multiplyCurrencies(
+    tokenToFiatRate = multiplyCurrencies(
       conversionRate,
       selectedTokenExchangeRate,
       { toNumericBase: 'dec' }
     )
   }
 
   return {
-    selectedAccount: getCurrentAccountWithSendEtherInfo(state),
-    accounts: accountsWithSendEtherInfoSelector(state),
+    ...state.metamask.send,
```

Comment:
i like to be more explicit in props mapper instead of spreading whole state. Make for easier debugging in the future